frontend/src/types.ts

```diff
@@ -896,9 +896,11 @@ export interface SessionRecordingsResponse {
     has_next: boolean
 }
 
+export type ErrorClusterSample = { session_id: string; input: string }
```

Comment:
I can't think of a better name but I'd love a clearer name for this at some point... that I can't think of a name might suggest it's my understanding and not the name though 🤔